seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-feishu/src/main/resources/META-INF/services/org.apache.seatunnel.spark.BaseSparkSource

```diff
@@ -0,0 +1,18 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+org.apache.seatunnel.spark.source.FeishuSheet
```

Comment:
It might be better to rename your plugin package to `org.apache.seatunnel.spark.feishu.source.FeishuSheet`.

Reply:
Sure. Done!

Reply:
This error should not relate with this PR.